app/controllers/courses/controllers.rb

```diff
@@ -0,0 +1 @@
+class Courses::Controllers < ApplicationController; end
```

Comment:
Added already, otherwise there will be an red new line symbol at the end of line.